### DISCUSSION TITLE
Anonymization (RTBF)?

### DISCUSSION THREAD

**Discussion by abador** [ory/kratos#1513] [Q&A] [answered]
There are some EU directives regarding the Right to be forgotten. 
The user should be able to start a process where all of the personal data collected by him can be changed so that the person can't be identified anymore ([link](https://en.wikipedia.org/wiki/Right_to_be_forgotten))
This might mean deleting the account or anonymization.
Is there a plan to implement such mechanisms in kratos?

**Comment by aeneasr** [comment, marked as answer]
#596 :)
